vru_use_case_scripts/plot_detected_objects

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python3
+
+from argparse import ArgumentParser
+from collections import defaultdict
+import csv
+import json
+from pathlib import Path
+import sys
+
+from matplotlib import pyplot as plt
+import numpy as np
+import pandas as pd
+from scipy.interpolate import splrep, BSpline
+
+
+def parse_positions_from_sdsm_csv(file_path: Path):
+    def get_timestamp(csv_row: list):
+        return round(float(csv_row[1]))
+
+    def get_json_payload(csv_row: list):
+        json_string = csv_row[8]
```

Comment:
I assume we are taking indexes to avoid column name changes impacting this logic? Is that accurate?